### ISSUE TITLE
Media Play/Pause Key not working

### ISSUE THREAD

**Issue by Ecpii** [th-ch/youtube-music#415]
After having the app open for a while, using the play/pause button on my keyboard will not pause tracks. Pressing this button will still show the on screen display, and the play/pause icon will sometimes toggle, but the music will continue playing.

[video here (with sound)](https://imgur.com/a/EO3jvxx)

**Plugins**:
- adblocker
- auto-confirm-when-paused
- discord
- in-app-menu
- navigation

**Settings**:
Auto-update
Resume last song when app starts
Tray: enabled + app hidden

**Version**: 1.13.0, installed from Chocolatey

**Comment by JsephGriff** [comment]
Same issue here, seems like it might have stemmed from a recent windows update, since it was working fine on the same version just weeks before.

**Comment by karanimaan** [comment]
I had the same issue on v 1.13.0, but it seems to work on v 1.12.2 for me.

**Comment by mrengy** [comment]
I am having the same issue on Mac OS 10.15.7 / YouTube Music 1.12.1 . 

Also, it used to work fine but the issue cropped up recently, and I haven't updated YouTube Music, so not sure what caused it.

**Comment by mrengy** [comment]
Also happening on YouTube Music 1.12.2.

On both 1.12.2 and 1.12.1, the behavior is erratic. Here's what sometimes happens (this has happened repeatedly): 

1. With YT Music app not in focus but playing, press the keyboard play/pause button: nothing happens. 
2. Click on the YT Music app (on the top of the window) to bring it in focus: music pauses. 
3. Press keyboard play/pause button: then works as expected, for a short while.
4. Click on another app to bring it in focus, press keyboard play/pause button: works as expected. 

Other times at step 4, the keyboard play/pause button doesn't work. I can't seem to isolate when it works versus when it doesn't work.

**Comment by Ecpii** [comment]
Yeah, I'm on Windows 10 version 21H1, and my testing has shown similar behavior to @mrengy 's described results - it doesn't work until it's in focus or recently focused.

Another thing I've noticed is that the button in the application itself will be in the wrong state after trying to pause the track - music will be playing and the button will be the "play" icon. Clicking it pauses the music for half a second, switches the button back to the pause icon, and continues playing it. After that, the button works as expected.

Additionally, although quite expected, trying to pause tracks through other controllers (like my earbuds) also doesn't work.

**Comment by Araxeus** [comment]
You could meanwhile setup custom hotkeys using the `shortcuts` plugin

here is a short explanation how to do it - https://github.com/th-ch/youtube-music/issues/235#issuecomment-821798596

(this is not a fix for this problem, just a workaround - will work better if #243 is merged)

**Comment by mrengy** [comment]
Seems to be working nicely in v 1.14.0. This made my day.
